datasets/mx/tax_offender/crawler.py

```diff
@@ -0,0 +1,124 @@
+import csv
+from typing import List
+
+from zavod import Context, helpers as h
+
+URLs = [
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Cancelados.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/ReduccionArt74CFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonadosart146BCFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonadosart21CFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/CondonadosporDecreto.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonados_07_15.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Cancelados_07_15.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Retornoinversiones.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Exigibles.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Firmes.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/No%20localizados.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Sentencias.csv",
+]
```

Comment:
can we take these from the HTML automatically? Something like all urls from the table with an allow-list and deny list of labels, so we exclude `Documento técnico y normativo` automatically, and get a warning when something that's neither in the allow list nor the deny list shows up? That way we can make 100% sure we don't miss new files, changed links, or accidentally add files that shouldn't be included automatically. a `context.log.warning("Unknown file found", label=label, url=url)`

Reply:
I tried fetching the HTML using the `fetch_html` but it needs some kind of cookie, I can try to see if I can find a workaround if you want.

Reply:
aha let's use http://omawww.sat.gob.mx/cifras_sat/Paginas/datos/ListCompleta69.html for the data URL

Reply:
It worked!